packages/minifiers-css/minifiers.js

```diff
@@ -110,7 +111,7 @@ var rewriteRules = function (rules) {
         basePath = "/";
 
     _.each(rule.declarations, function(declaration, declarationIndex) {
-      var parts, resource, absolutePath, quotes, oldCssUrl, newCssUrl;
+      var parts, resource, absolutePath, relativeToMergedCss, quote, oldCssUrl, newCssUrl;
```

Comment:
might be worth breaking down into two lines to 1) logically group the variables and 2) make the lines max 80 chars long